src/AutoMapper/PropertyMap.cs

```diff
@@ -41,7 +41,7 @@ public PropertyMap(PropertyMap inheritedMappedProperty, TypeMap typeMap)
         public LambdaExpression CustomExpression { get; private set; }
         public MemberInfo CustomSourceMember { get; set; }
         public bool UseDestinationValue { get; set; }
-        public bool ExplicitExpansion { get; set; }
+        public bool ExplicitExpansion { get { return _explicitExpansion.GetValueOrDefault(); } set { _explicitExpansion = value; } } private bool? _explicitExpansion; 
```

Comment:
Anyway, it is confusing pretty much with or without working with EF. 

If inheritance can be done within some other scope, my issue could be added to that scope as a related issue. If this is going to be fixed in that scope and the code in my PR is not much helpful, the PR can be just declined, or I'll revoke it, if GitHub allows something like this, If the issue is found irrelevant, we can just close it, leaving this confuse intact.

Thanks, Oleksii.